charts/openebs/openebs-control-plane-pod-delete/engine.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: openebs-control-plane-chaos
+  namespace: openebs
+spec:
+  # # It can be true/false
+  annotationCheck: 'false'
+  auxiliaryAppInfo: ''
```

Comment:
pls provide comment for sample input like `#ex. values: ns1:name=percona,ns2:run=nginx `